packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -896,6 +896,8 @@ new tasks.SageMakerCreateTransformJob(this, 'Batch Inference', {
 
 ```
 
+The `instanceType` field can be a parameter. If you set it to `sfn.JsonPath.stringAt('$.InstanceType')`, it will be rendered as `'InstanceType.$': '$.InstanceType'` in the final CloudFormation template.
```

Comment:
please use line breaks at 80/100 chars.

I think what you intend to say here is that it can be JsonPath from the state input rather than it can be a `parameter`.

Reply:
Thanks, fixed